gpt4all-chat/qml/ChatView.qml

```diff
@@ -1354,9 +1354,6 @@ Rectangle {
                             ToolTip.text: Accessible.description
 
                             onClicked: {
-                                var index = Math.max(0, chatModel.count - 1);
-                                var listElement = chatModel.get(index);
-                                listElement.stopped = true
```

Comment:
We should leave this commented out with a FIXME stating that it seems to be broken as of v3.0.0 and that ChatModel::get has since been removed.